manubot/pandoc/tests/test_cite_filter/output.txt

```diff
@@ -10,52 +10,42 @@ Defining citekeys with the link reference syntax [3,4,5].
 
 Citing pre-defined citekey aliases [6,7].
 
-Citing 8 whose metadata is already in pandoc’s reference metadata.
+Citing [8] whose metadata is already in pandoc’s reference metadata.
 
 Citing works whose reference metadata is provided via --bibliography [9,10].
 
 References
 
 1. Orthodontic treatment with tooth transplantation for patients with cleft lip and palate.
-Kotaro Tanimoto, Tamami Yanagida, Kazuo Tanne
-The Cleft palate-craniofacial journal : official publication of the American Cleft Palate-Craniofacial Association (2010-09) https://www.ncbi.nlm.nih.gov/pubmed/20170387
+Kotaro Tanimoto, Tamami Yanagida, Kazuo Tanne The Cleft palate-craniofacial journal : official publication of the American Cleft Palate-Craniofacial Association (2010-09) https://www.ncbi.nlm.nih.gov/pubmed/20170387
 DOI: 10.1597/08-134 · PMID: 20170387
 
 2. Lung cancer incidence decreases with elevation: evidence for oxygen as an inhaled carcinogen
-Kamen P. Simeonov, Daniel S. Himmelstein
-PeerJ (2015-01-13) https://doi.org/98p
+Kamen P Simeonov, Daniel S Himmelstein PeerJ (2015-01-13) https://doi.org/98p
```

Comment:
Will have to look into the missing newline between author and venue in md output. Related to https://github.com/jgm/pandoc/issues/6921